src/CMakeLists.txt

```diff
@@ -211,8 +213,14 @@ set(FIRMWARE-DRIVER-SOURCES
 set(FIRMWARE-DRIVER-SOURCES ${FIRMWARE-DRIVER-SOURCES} PARENT_SCOPE)
 
 set(FIRMWARE-U2F-SOURCES
+  ${CMAKE_SOURCE_DIR}/src/fido2/ctap.c
```

Comment:
did you modify those files in any way?

would be great to move them to external/ instead and compile them as a lib, like `ctaes` for example.